rmgpy/ml/__init__.py

```diff
@@ -1,6 +1,3 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
```

Comment:
Did you remove all of the shebangs intentionally? Are they not necessary in Python 3?

Reply:
The coding specification is not necessary in Python 3 because that is now the default.

I removed the first line as well because the modules aren't meant to be run as scripts and Python doesn't need that line if you're importing a module. That line is only useful if you plan to run a Python file without telling the command line explicitly that you're using Python. Most RMG files have that first line but unless they're scripts, none of those files need it. We could consider removing it everywhere. Or would you rather I keep it here to be consistent with the rest of RMG?

Reply:
In that case, I think we can remove them everywhere. That can be a separate PR though.

Reply:
Should I keep it here then?